app/views/shared/_base_page.html.erb

```diff
@@ -17,7 +17,7 @@
           <%= yield %>
         </div>
       </div>
-      <%= render 'shared/publication_metadata', :publication => publication %>
+      <%= render 'shared/publication_metadata', :publication => content_item %>
```

Comment:
Is this going to work? We're passing it in as the local "publication" but then using it in the file below as the local "content_item".

Reply:
Good spot! It works because local "publication" is not being used. I'll remove the local as it's not needed.